### PR TITLE
Added new blog post

### PR DESCRIPTION
**Pull request by stevemac007**

### CODE REVIEW

File: _posts/2017-01-18-youre-doing-it-wrong-if.markdown [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - development
+tags:
+    - best-practice
+author: Steve Mactaggart
+images:
+
+excerpt:
+    We are all looking to learn from our failures and make things better, but sometimes it better to communicate what is right by eliminating what is wrong.<br/>
+    <br/>
+    <i>"When you have eliminated whats wrong, whatever remains, however improbable, must be right".</i>
```
Comment (labrute974) [inline _posts/2017-01-18-youre-doing-it-wrong-if.markdown]: what's wrong

File: _posts/2017-01-18-youre-doing-it-wrong-if.markdown [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+I've been around for a while, worked through different teams, across different industries and companies.  
+Over time I have learnt a lot of lessons, some the easy way, most the hard way.
+
+While the failures have all been different, their learnings have regularly overlapped and now form a set of practices that guide me on a daily basis. I've tried lots of different ways to collect these patterns, and for a long time I was focused on passing on what the best practices were.  But they never seemed to fit.
```
Comment (labrute974) [inline _posts/2017-01-18-youre-doing-it-wrong-if.markdown]: that guides me

File: _posts/2017-01-18-youre-doing-it-wrong-if.markdown [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+> "When you have eliminated **whats wrong**, whatever remains, however improbable, **must be right**".
+
+It's can be really liberating to take this approach, by defining what not to do you _don't have to be right_, you __only don't have to be not wrong__.  It might sound counter intuitive, but I think you'll find that it's much easier to exclude the things that have caused pain and still maintain flexibility to foster innovation.
+
+So here is my list, these are some of the core rules that I use to help differentiate between what we should and shouldn't be doing.
+
+## You're doing it wrong if...
+
+1. the first time you are something is in production
```
Comment (labrute974) [inline _posts/2017-01-18-youre-doing-it-wrong-if.markdown]: you are *doing* something

File: _posts/2017-01-18-youre-doing-it-wrong-if.markdown (hunk 2 above)
Comment (stevemac007) [inline _posts/2017-01-18-youre-doing-it-wrong-if.markdown]: guide is right there, they guide me on a daily basis, not guides me..

File: _posts/2017-01-18-youre-doing-it-wrong-if.markdown (hunk 3 above)
Comment (stevemac007) [inline _posts/2017-01-18-youre-doing-it-wrong-if.markdown]: thx

**Pull request merged**